pyfar/classes/filter.py

```diff
@@ -434,33 +442,50 @@ def init_state(self, cshape, state='zeros'):
                 new_state[idx, ...] = spsignal.lfilter_zi(coeff[0], coeff[1])
         super().init_state(state=new_state)
 
+    def impulse_response_length(self):
```

Comment:
ir length sounds like seconds, but not like samples, maybe we can rename to `impulse_response_samples` or similar

Reply:
I would be fine with changing the name. Lets maybe find the final name in the weekly. Otherwise it might be a ping pong between different names...

Reply:
`impulse_response_samples` to me sounds like the value of the samples, not the length of the IR in samples.
samples is only the unit used to quantify the length. So length would be fitting better in my opinion.

Reply:
We decided to call this `minimum_impulse_response_length`